containers/DevMining.ts

```diff
@@ -53,11 +58,13 @@ const useDevMiningCalculator = () => {
       })
       .then((rewards) => setRewards(new Map(rewards)))
       .catch((err) => {
+        setError(err);
         console.error(err, "Error calculating dev mining rewards");
       });
-  }, [provider, totalRewards, empWhitelist]);
+  }, [provider, totalRewards, empWhitelist, contract]);
```

Comment:
allows user to refresh dev mining calculations if there was an error by switching contracts rather than reloading whole page